### PR TITLE
Adds "exo version" command

### PR DESCRIPTION
**Pull request by martinjaime**
Addresses #174 

@kevgo, @trushton, @hugobho, @alexdavid

### CODE REVIEW

**Comment by martinjaime** [comment]
I'm not sure why, but CircleCI is saying that the bash command `which` is missing.

**Comment by martinjaime** [comment]
Tests pass! Circle said that Ubuntu builds were degraded, so I'm not sure if they fixed that, or if they passed because I ran without cache.

File: features/tutorial.feature [hunk 1]
```diff
@@ -15,6 +15,11 @@ Feature: Following the tutorial
     you always have to run the whole feature.
 
 
+  Scenario: verify that exo commands can be run by running "exo version"
+    When starting "exo version" in the terminal
+    Then waiting until I see "Exosphere version" in the terminal
```
Comment (kevgo) [inline features/tutorial.feature]:
```cucumber
Then I see "Exosphere version" in the terminal
```

You might have to adjust the regex of the step to allow for both "waiting until I see" and "I see".

**Comment by martinjaime** [comment]
@kevgo. updated the regex to accept both expressions.

**Pull request closed without merging**